### DISCUSSION TITLE
Recharts on Climate Change Tracker

### DISCUSSION THREAD

**Discussion by alexcybernetic** [recharts/recharts#3368] [Show and tell]
👋 Hey folks, we are using Recharts on Climate Change Tracker! 
Thank you all for doing this great work! We'd be happy to contribute in future to Recharts! 💪
Any questions, feedback and improvement suggestions are very welcome. 💫

Check out [https://climatechangetracker.org](https://climatechangetracker.org)

Light mode
<img width="1575" alt="Screenshot 2023-02-14 at 11 34 35" src="https://user-images.githubusercontent.com/7684726/218710811-a234c263-9456-42a7-9116-5beedd772d6a.png">

Dark mode
<img width="1363" alt="Screenshot 2023-02-14 at 17 12 09" src="https://user-images.githubusercontent.com/7684726/218794710-95b2b216-0517-4a13-b861-489bc5f532e2.png">

All responsive
<img width="482" alt="Screenshot 2023-02-14 at 17 12 38" src="https://user-images.githubusercontent.com/7684726/218794755-64805b17-c74b-4c25-aa71-f90003b008ab.png">
<img width="490" alt="Screenshot 2023-02-14 at 17 13 13" src="https://user-images.githubusercontent.com/7684726/218794766-532d4c3e-c893-44e1-bc3d-3bdb60e91db1.png">

**Comment by ckifer** [comment]
This is beautiful! Thank you for sharing!

Please please, contributions and active contributors are always welcome.

**Comment by nikolasrieble** [comment]
Amazing work. I would love to link this on the website as a great example project. Would you be fine with that? @a2x

**Reply by alexcybernetic** [reply]
Thank you @nikolasrieble, very fine!

**Reply by alexcybernetic** [reply]
@nikolasrieble I've edited and uploaded the logo
https://github.com/recharts/recharts/wiki/Who-uses-recharts%3F
